src/index.js

```diff
@@ -1,40 +1,71 @@
 import './style.css';
+import Features from './module/functionality.js';
+import Storage from './module/localstorage.js';
 
-const container = document.querySelector('.todo-listing');
-
-const todoList = [
-  {
-    description: 'Read my books',
-    completed: true,
-    index: 3,
-  },
-  {
-    description: 'Study data structures',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'Learn new coding concepts',
-    completed: false,
-    index: 2,
-  },
-
-];
-
-const renderTask = () => {
-  const sortedList = todoList.sort((item1, item2) => item1.index - item2.index);
-
-  for (let i = 0; i < sortedList.length; i += 1) {
-    container.innerHTML += `
-    <div class="task-container">
-      <form class="task-form">
-        <input type="checkbox" class="checkbox">
-      </form>
-    <p class="task-description">${sortedList[i].description}</p>
-  </div>
-  <hr>
-    `;
+const ulContainer = document.querySelector('.list-content');
+
+const renderTasks = () => {
+  let taskList;
+
+  if (Storage.getLocalStorage() === null) {
+    taskList = [];
+  } else {
+    taskList = Storage.getLocalStorage();
   }
+
+  let content = '';
+
+  taskList.forEach((task, id) => {
+    content += `
+    <li class="list-items">
+    <div class="render-div">
+      <input class="check" type="checkbox">
+      <input class="task-description" id="task${id}" value=${task.description} />
+    </div>
+    <div class="icon-content">
+      <i id="removeTask${id}" class="sective fa-solid fa-trash-can"></i>
+    </div>
+  </li>`;
+  });
+
+  ulContainer.innerHTML = content;
+
+  taskList.forEach((task, index) => {
+    const removeTask = document.getElementById(`removeTask${task.index}`);
+    if (removeTask) {
+      removeTask.addEventListener('click', () => {
+        Features.removeTask(index);
+        removeTask.remove();
+        renderTasks();
+      });
+    }
+  });
```

Comment:
- [ ] right now it's updating the index after deleting an item but when I tried to delete the last task it won't so please make sure to fix that so that you complete the requirements